test/test_run.py

```diff
@@ -46,6 +49,35 @@ def test_run_tree_info(self):
             gokart.run()
         self.assertTrue(gokart.make_tree_info(_DummyTask(param='test')), tree_info.output().load())
 
+    @patch('gokart.make_tree_info')
+    def test_try_to_send_event_summary_to_slack(self, make_tree_info_mock: MagicMock):
+        event_aggregator_mock = MagicMock()
+        event_aggregator_mock.get_summury.return_value = f'{__name__}._DummyTask'
+        event_aggregator_mock.get_event_list.return_value = f'{__name__}._DummyTask:[]'
+        make_tree_info_mock.return_value = 'tree'
+
+        def get_content(content: str, **kwargs):
+            self.output = content
+        slack_api_mock = MagicMock()
+        slack_api_mock.send_snippet.side_effect = get_content
+
+        cmdline_args = [f'{__name__}._DummyTask', '--param', 'test']
+        with patch('gokart.slack.SlackConfig.send_tree_info', True):
+            _try_to_send_event_summary_to_slack(slack_api_mock, event_aggregator_mock, cmdline_args)
+        expects = os.linesep.join(['===== Event List ====', event_aggregator_mock.get_event_list(), os.linesep])\
+            + os.linesep.join(['==== Tree Info ====', 'tree'])
```

Comment:
[nits]
Following code might be simpler.
```
os.linesep.join(['===== Event List ====', event_aggregator_mock.get_event_list(), os.linesep, '==== Tree Info ====', 'tree'])
```

Reply:
Thank you! I fixed it!!!!